src/components/LegacyQueryEditor/RawQueryEditor.tsx

```diff
@@ -41,6 +41,15 @@ interface Worker {
 // that includes fixes required for auto-completion to work.
 // Remove this code once Grafana 8.5 is the minimal version supported
 function gtGrafana8_5() {
+  const version = config.buildInfo.version
+  const isValid = valid(config.buildInfo.version)
+  // Assume that a security release will be of the form 'w.x.y.z' - Pre releases of the form 'w.x.y-pre.z' are already valid
+  const isSecurityRelease = version.split('.').length > 3
+  if (!isValid && isSecurityRelease) {
+    // Coerce will drop the z in 'w.x.y.z' leaving just the major-minor-patch version
+    const coercedValue = coerce(version)
+    return coercedValue ? gte(coercedValue, '8.5.0') : null;
+  }
   return valid(config.buildInfo.version) && gte(config.buildInfo.version, '8.5.0');
```

Comment:
Nit: could return `isValid` here instead of re-running `valid`. Could also pass `version` instead of the full object address again.